code/modules/projectiles/guns/specialist.dm

```diff
@@ -578,18 +546,24 @@
 		return
 	fire_grenade(target, gun_user)
 	gun_user.hud_used.update_ammo_hud(gun_user, src)
+	return TRUE
 
+/obj/item/weapon/gun/launcher/attackby(obj/item/I, mob/user, params)
+	. = ..()
+	if(!istype(I, /obj/item/explosive/grenade))
+		return ..()
+	if(length(grenades) >= max_grenades)
+		to_chat(user, span_warning("[src] cannot hold more grenades!"))
+		return
+	if(!user.transferItemToLoc(I, src))
+		return
+	grenades += I
+	playsound(user, 'sound/weapons/guns/interact/shotgun_shell_insert.ogg', 25, 1)
+	to_chat(user, span_notice("You put [I] in [src]."))
+	to_chat(user, span_info("Now storing: [grenades.len] / [max_grenades] grenades."))
+	user.hud_used.update_ammo_hud(user, src)
 
-//Doesn't use most of any of these. Listed for reference.
-/obj/item/weapon/gun/launcher/m92/load_into_chamber()
-	return
-
-
-/obj/item/weapon/gun/launcher/m92/reload_into_chamber()
```

Comment:
put this and load_into_chamber() up into the /launcher typepath and remove the ones for the 81